trail/upload/process_uploads/header_standardizer.py

```
@@ -323,3 +328,36 @@ def standardizeWcs(self, hdu=None):
 
         wcs = models.Wcs(**self._computeStandardizedWcs(header, dimX, dimY))
         return wcs
+
+    def getWcs(self, path_to_file):
+        """Given a fits file it will process and send to astrometry.net
+        where it will obtain the WCS data for the file if it is able to.
+        If not then it will return an empty dictionary
+        (this can be altered to whatever needs to be used for)
+
+        Parameters
+        ----------
+        path_to_file : `string`
+            path to the fits file
+
+        Returns
+        -------
+        header : `dict`
+            returns the header with the WCS data in it if it was found.
+            If an error occurred or no solution is found then returns an empty dictionary.
+
+        Notes
+        -----
+        Currently this just ignores errors in the process.
+        It might be better to just raise the errors and let whatever the other function is to handle it.
+        """
+        try:
+            header = ast.solve_from_image(path_to_file, False)
+            # return that there was an error
+        except TimeoutError:
```

Don't silence errors here. Let all the errors propagate all the way up to [`FitsProcessor.process`](https://github.com/dirac-institute/trailblazer/blob/main/trail/upload/process_uploads/fits_processor.py#L295) where we should, eventually, try to handle them or at the very lease propagate back to the user in the [`views.py`](https://github.com/dirac-institute/trailblazer/blob/main/trail/upload/views.py).

At best what we could do here is to define and raise a new exception like `class StandardizeWcsException(Exception)` and then `raise StandardizeWcsException(msg, attrs) from err`. 

What you should also do is add a `Raises` section to the docstring, instead of the `Notes` section, documenting what types of errors the function might raise.